lua/plenary/functional.lua

```diff
@@ -2,7 +2,7 @@ package.loaded['plenary.functional'] = nil
 
 local f = {}
```

Comment:
Should we add some `curry`/`uncurry` functions? I know at that point we're beginning to overlap more with [lua-fun](https://github.com/luafun/luafun), which we've talked about on Gitter, but maybe could still add those functions regardless, at least for now?

Reply:
let's tackle that on another day.